app/assets/stylesheets/argo/impersonate_groups.scss

```diff
@@ -0,0 +1,12 @@
+.auth-groups {
+  margin: 0 auto;
+  width: 500px;
+}
+
+.navbar-nav > li > a.impersonating {
```

Comment:
I could not get a way from these style violations... needing to overwrite bootstrap's default.